code/modules/clothing/head/misc.dm

```diff
@@ -280,5 +280,5 @@
 
 /obj/item/clothing/head/crown/fancy
 	name = "magnificent crown"
-	desc = "A crown worn by only the highest emperors of the land."
+	desc = "A crown worn by only the highest emperors of the <s>land</s> space."

```

Comment:
Mentally including the strikethrough is the same as that text not existing.
leaving just "the space" which as I've said, is stupid and wrong.